### PR TITLE
Add overflow-wrap: break-word for Safari

### LINKED ISSUE #10289: Long strings break the layout of the homepage/detail pages on iOS
I've noticed that long unbroken strings for addon names don't look good - on FF for iOS 

looks truncated on iOS

detail page on iOS

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #10289

### CODE REVIEW

File: src/amo/components/App/styles.scss [hunk 1]
```diff
@@ -31,6 +31,9 @@ textarea {
 *::before,
 *::after {
   box-sizing: inherit;
+  // This is needed for Safari, which does not support `anywhere`.
+
```
Comment (willdurand) [inline src/amo/components/App/styles.scss:35]:
```suggestion
```

**Review by willdurand — COMMENTED** [review]
I am wondering if we can suggest a blank line removal :p

File: src/amo/components/App/styles.scss (hunk 1 above)
Comment (championshuttler) [reply src/amo/components/App/styles.scss:35]: heh wait

Comment (championshuttler) [reply src/amo/components/App/styles.scss:35]: i followed the comment below that has also one line space :P

Comment (willdurand) [reply src/amo/components/App/styles.scss]: whaaaat?

Comment (willdurand) [reply src/amo/components/App/styles.scss]: ah I see now :(

**Review by willdurand — APPROVED** [review]

**Pull request merged**